### DISCUSSION TITLE
Use SQLITE as datasource

### DISCUSSION THREAD

**Discussion by gryckelynck** [Budibase/budibase#2053] [Ideas]
Do you have plan to use Sqlite database as datasource?

**Comment by mike12345567** [comment]
Hi @gryckelynck - converting this to a feature request so that others can vote on it!

**Comment by christophberger** [comment]
I would love to see SQLite as a datasource.
- It is dead easy to manage
- The database is a simple file, to backup the DB, simply make a copy
- Simple streaming backup to S3-compatible storage with Litestream
- If used as in-app database, no network latencies can slow down DB operations
- There are several options available for creating a cluster of distributed SQLite databases (LiteFS, dqlite, rqlite,...)

**Comment by reignmaker** [comment]
Any news on adding SQLite as a datasource?

**Comment by epicwhale** [comment]
Would really love to see this as a data source!! +1000

**Comment by MitPitt** [comment]
+1

**Comment by andrej1919** [comment]
+1

**Comment by victor9000** [comment]
So... this is the most voted feature request, any luck?

**Comment by th-schall** [comment]
I would love to see this feature! A local SQLite database file as data source may easily enable simple personal offline data management apps. I would have  quite a few use cases to implement with that already.
